### PR TITLE
fixes for integration into cloud explorer

### PR DESCRIPTION
**Pull request by lexoyo**

### CODE REVIEW

**Comment by lexoyo** [comment]
I must say it is not well tested, I am in the middle of a cloudy (explorer) mess

File: lib/unifile-fs.js [hunk 1]
```diff
@@ -32,6 +33,19 @@ class FsConnector {
     this.name = NAME;
   }
 
+  resolvePath(path) {
+    // is the path alreay resolved
+    if(path.startsWith(Os.homedir()) && Path.resolve(path).startsWith(Os.homedir()))
```
Comment (JbIPS) [inline lib/unifile-fs.js]: I think using `Path.isAbsolute()` would be a better option.

File: lib/unifile-fs.js [hunk 2]
```diff
@@ -32,6 +33,19 @@ class FsConnector {
...
+      return path;
+    // or we resolve it in the user ~
+    const secure = Path.resolve(Os.homedir(), path);
```
Comment (JbIPS) [inline lib/unifile-fs.js]: I understand the concern here but I don't think it should be Unifile who restricts access. 

I see two options here:
* no control, the program using the library will set the limits 
* a 'safe path' given as an argument to the constructor of FS connector. 

In the exemple of Silex, I don't think you want user to meddle in Silex home directory 😊

**Comment by JbIPS** [comment]
Merged with https://github.com/silexlabs/unifile/commit/24547096c2e1462e45868712ce311617b73e3101 and https://github.com/silexlabs/unifile/commit/edebb035c6586f6f267c87e96c7e64aea108184f

**Pull request closed without merging**